devenv/README.md

```diff
@@ -2,11 +2,43 @@
 
 A Developer Environment (or devenv for short) is a workspace for developers to make changes without breaking anything in a live environment. 
 
-In order to foster contribution for the Netdata Agent and it's various collectors, we created an image that is a devenv on it'own, a fully containerized environment where it has everything a developer needs to develop improvements on the Netdata Agent itself or it's collectors. 
+In order to foster contribution for the Netdata Agent and it's various collectors, we created an image that is a devenv on it's own, a fully containerized environment where it has everything a developer needs to develop improvements on the Netdata Agent itself or it's collectors.
 
-# Instructions 
+We have included all the dependencies that you need to build the Netdata Agent and it's collectors. Moreover, we have included helpful tools such as `python pip` and `flake8`. 
 
-- Clone this repository 
-- `cd` into the *devenv/docker* directory
-- run `docker build -t netdata_devenv:latest` and `docker run -p 19999:19999 devenv`
-  
\ No newline at end of file
+You only need to download your forked repositories and you can begin developing right away!
+
+# Dockerfile
+
+Inside this repository you can find the Dockerfile which is used to create the Docker Container. It is automatically published on [DockerHub](https://hub.docker.com/r/netdata/devenv), using a CI/CD pipeline based on GitHub Actions. 
+
+# How to use the Devenv 
+
+This devenv makes use of:
+- [VS Code](https://code.visualstudio.com/), one of the most used open source editors on the Internet.
+- [Remote Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) VS Code extension.
+- [Docker](https://www.docker.com/).
```

Comment:
I would specifically call out that using VSCode is optional. Yes, we can directly integrate with it, but it0s not required to use the development environment.